### PR TITLE
Corrected definition for stats.js (r14)

### PR DESCRIPTION
**Pull request by nicky-lenaers**
Updated the definition file according to best practices by exporting the module under the name `"stats"`.

### CODE REVIEW

**Comment by vvakame** [comment]
*stats/stats.d.ts*

to author(@gregolai). could you review this PR?
:+1: or :-1:?

check list

* [ ] pass the Travic-CI test?

**Comment by nicky-lenaers** [comment]
@vvakame Any way you could review this? It's been some time and I expect @gregolai not to answer anytime soon. It's a tiny PR so shouldn't be much of a hassle!

File: stats/stats.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for Stats.js r12
+// Type definitions for Stats.js r14
 // Project: http://github.com/mrdoob/stats.js
-// Definitions by: Gregory Dalton <https://github.com/gregolai>
```
Comment (vvakame) [inline stats/stats.d.ts]: keep original author name. like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3

**Comment by vvakame** [comment]
I reviewed your PR.

---
note
https://github.com/mrdoob/stats.js/blob/master/src/Stats.js#L182

**Comment by RyanCavanaugh** [comment]
Hello and thank you for your contribution!

Due to an excessively long queue of pull requests that have become stale, we are "declaring bankruptcy" and closing all PRs opened before May 1, 2016. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

Going forward, we are committing to review or merge all PRs on a regular basis so this bankruptcy will not occur again. We apologize for the incovenience and hope you will continue to contribute to DefinitelyTyped in the future.

Thanks
The TypeScript and DefinitelyTyped teams

**Pull request closed without merging**